dao/src/main/resources/db/migration/V89__addProjectDescription.sql

```diff
@@ -0,0 +1,3 @@
+-- Add a description column to the projects table because project descriptions were added to ORT.
+
+ALTER TABLE projects ADD COLUMN description TEXT NOT NULL DEFAULT '';
```

Comment:
In the upstream model, the description is optional.
Why is the description required in the DB?

Reply:
`Project.description` is not nullable in the ORT model.